folly/experimental/TestUtil.cpp

```diff
@@ -28,6 +28,8 @@
 #include <folly/FileUtil.h>
 #include <folly/String.h>
 
+extern char** environ;
```

Comment:
This needs to be wrapped in a `#ifndef _MSC_VER`, because MSVC defines this differently, and doing this will break it.